### ISSUE TITLE
Feature Idea: AKAI Sample CDs 

### ISSUE THREAD

**Issue by mrglennjones** [schollz/mx.samples#8]
Hi, 
I love this script a lot, thank you!

I had an an idea.

What if it could read and mount AKAI Sample CD Images from a remote internet source or USB Memory stick/SSD?

**Comment by schollz** [comment]
yeah it may be possible! do you have any example sources that I could check out?

**Comment by mrglennjones** [comment]
yes try here https://vintagemusicsamples.wordpress.com/

**Comment by mrglennjones** [comment]
here's some info on detecting the img filetype and mounting it...  https://unix.stackexchange.com/questions/82314/how-to-find-the-type-of-an-img-file-and-mount-it

**Comment by mrglennjones** [comment]
further to this I thought it'd be nice to utilize personal cloud storage services for storing the AKAI imgs like google drive etc.
